tests/searchcommands/test_searchcommands_app.py

```diff
@@ -377,8 +379,8 @@ def _load_chunks(self, ifile):
             body_length = int(match.group('body_length'))
             body = ifile.read(body_length) if body_length > 0 else ''
 
-            if len(chunks) == 0:
-                self.assertEqual(ifile.readline(), '\n')  # the getinfo exchange protocol requires this
+            # if len(chunks) == 0:
```

Comment:
How come this is commented out? I'd prefer to not see new commented-out code committed to the repo.